docs/sources/reference/components/otelcol/otelcol.processor.batch.md

```diff
@@ -16,9 +16,9 @@ title: otelcol.processor.batch
 Batching improves the compression of data and reduces the number of outgoing network requests required to transmit data.
 This processor supports both size and time based batching.
 
-We strongly recommend that you configure the batch processor on every {{< param "PRODUCT_NAME" >}} that uses OpenTelemetry (otelcol) {{< param "PRODUCT_NAME" >}} components.
-The batch processor should be defined in the pipeline after the `otelcol.processor.memory_limiter` as well as any sampling processors.
-This is because batching should happen after any data drops such as sampling.
+Grafana Labs strongly recommends that you configure the batch processor on every {{< param "PRODUCT_NAME" >}} that uses OpenTelemetry (otelcol) {{< param "PRODUCT_NAME" >}} components.
```

Comment:
This recommendation is changing upstream, the recommendation now is to use the exporter's sending_queue for batching in most cases. However I don't think even upstream has this well documented.